docs/developers/dvm-integration.md

```diff
@@ -3,25 +3,27 @@ title: Integrating with the DVM
 sidebar_label: Integrating with the DVM
 ---
 
+**IMPORTANT NOTE**: In most cases, it is better to integrate with the Optimistic Oracle rather than integrating with the DVM directly. See the [Optimistic Oracle tutorial](developers/optimistic-oracle-integration.md) for details.
+
 ## Toy Integration Example
 
 This example will set up a `DepositBox` contract which custodies a user’s ERC-20 token balance.
-On a local testnet blockchain, the user will deposit ETH into the contract and withdraw ETH corresponding to a desired USD amount.
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH corresponding to a desired USD amount.
```

Comment:
In the rest of the docs, we call this WETH instead of wETH. I'm kind of impartial here, but generally think WETH is more pleasing and correct.

Reply:
I was going with the capitalization promoted by the dev team: https://weth.io/

If it's all the same to you, I'd prefer to stick with wETH since it's also what's in the demo script, but if WETH is preferable just for docs consistency, it's not a big change.

Reply:
I stand corrected! Just always see it capitalized I guess. To match the demo script, I'm fine with wETH